# HG changeset patch
# User Stefan Mahr <dac922@gmx.de>
# Date 1383430189 -3600
#      Sat Nov 02 23:09:49 2013 +0100
# Node ID 8b22a9215ec246a4d17a2cd7b3a034d4b16ceffd
# Parent  efa4572997baf0f490d9d9ce4859d1bf1da6d57c
jit compiler: Fix uninitialized variable warning

* libinterp/corefcn/pt-jit.cc (jit_convert::jit_convert):
Initialize variable retval.

diff --git a/libinterp/corefcn/pt-jit.cc b/libinterp/corefcn/pt-jit.cc
--- a/libinterp/corefcn/pt-jit.cc
+++ b/libinterp/corefcn/pt-jit.cc
@@ -161,17 +161,17 @@ jit_convert::jit_convert (octave_user_fu
   jit_value *return_value = 0;
   bool all_breaking = false;
   if (fcn.is_special_expr ())
     {
       tree_expression *expr = fcn.special_expr ();
       if (expr)
         {
           jit_variable *retvar = get_variable ("#return");
-          jit_value *retval;
+          jit_value *retval = 0;
           try
             {
               retval = visit (expr);
             }
           catch (const jit_break_exception&)
             { }
 
           if (breaks.size () || continues.size ())
